packages/boot/tools/supports.ts

```diff
@@ -367,6 +368,50 @@ export const makeSwingsetTestKit = async (
                   icaMocks.startChannelOpenInit.channelOpenAck(obj),
                 );
                 return undefined;
+              case 'sendPacket':
+                switch (obj.packet.data) {
+                  case 'eyJ0eXBlIjoxLCJkYXRhIjoiQ2xVS0l5OWpiM050YjNNdWMzUmhhMmx1Wnk1Mk1XSmxkR0V4TGsxelowUmxiR1ZuWVhSbEVpNEtDMk52YzIxdmN6RjBaWE4wRWhKamIzTnRiM04yWVd4dmNHVnlNWFJsYzNRYUN3b0ZkV0YwYjIwU0FqRXciLCJtZW1vIjoiIn0=':
```

Comment:
this is pretty specific to particular tests. please make these exported constants so it's easy to see where they're used

Reply:
https://github.com/Agoric/agoric-sdk/pull/9195#discussion_r1554459215